rhel7/profiles/stig.profile

```diff
@@ -206,7 +205,6 @@ selections:
     - audit_rules_sysadmin_actions
     - audit_rules_privileged_commands_newgrp
     - audit_rules_privileged_commands_chsh
-    - audit_rules_privileged_commands_sudoedit
```

Comment:
This seems like a bug and probably shouldn't be removed.

Reply:
This one can be a bug since it got dropped between v1r4 and v2r1, similar to `audit_rules_login_events_tallylog`.